### PR TITLE
fix(pat-select2): Show predefined value for single-select widgets.

### PR DESCRIPTION
**Pull request by thet**
Single-select widgets did not show their predefined value. This is now fixed.

fixed:

image

not fixed:

image

- [ ] I signed and returned the [Plone Contributor Agreement](https://plone.org/foundation/contributors-agreement), and received and accepted an invitation to join a team in the Plone GitHub organization.
- [ ] I verified there aren't any other open pull requests for the same change.
- [ ] I followed the guidelines in [Contributing to Plone](https://6.docs.plone.org/contributing/index.html).
- [ ] I successfully ran code quality checks on my changes locally.
- [ ] I successfully ran tests on my changes locally.
- [ ] If needed, I added new tests for my changes.
- [ ] If needed, I added [documentation](https://6.docs.plone.org/contributing/documentation/index.html) for my changes.
- [ ] I included a [change log entry](https://6.docs.plone.org/contributing/index.html#contributing-change-log-label) in my commits.

-----

If your pull request closes an open issue, include the exact text below, immediately followed by the issue number. When your pull request gets merged, then that issue will close automatically.

Closes #

### CODE REVIEW

File: src/pat/select2/select2.test.js [hunk 1]
```diff
@@ -100,6 +100,31 @@ describe("Select2", function () {
         expect(select2.options.ajax.url).toEqual("select2-users-vocabulary");
     });
 
+    it("renders the preset value of a single-select vocabulary widget", async function () {
+        document.body.innerHTML = `
+            <input class="pat-select2"
```
Comment (petschki) [inline src/pat/select2/select2.test.js:105]: Shouldn't that be a `<select>` if its not multiple values ? 

I wonder why this is needed, because the `language` field actually works with predefined values: https://github.com/plone/plone.app.dexterity/blob/master/src/plone/app/dexterity/behaviors/metadata.py#L110

**Review by petschki — CHANGES_REQUESTED** [review]

File: src/pat/select2/select2.test.js (hunk 1 above)
Comment (thet) [reply src/pat/select2/select2.test.js:105]: A‌ `<select>` field cannot use AJAX‌ remote data.

And for the timezone selection fields I do not want to use pre-populated list of options - because it can be huge - but a  text field where you can enter manually (if JS is off) or search/select via AJAX.

**Comment by thet** [comment]
@petschki

Btw. I have added a test showing that the `<select>` field isn't broken with this PR.
But thanks for the fixes in the querystringwidget!

File: src/pat/select2/select2.test.js (hunk 1 above)
Comment (petschki) [reply src/pat/select2/select2.test.js:105]: that makes totally sense

**Review by petschki — APPROVED** [review]
LGTM
